Style guide edits (#4895)
n8n-io/n8n-docs: docs/contribute/style-guide-for-n8n-docs.md

For the Markdown representation of every available block type, see the [GitBook documentation](https://gitbook.com/docs/creating-content/blocks).
{% endhint %}

### Headings

Write headings as plain Markdown (`## Heading text`), in sentence case. GitBook generates a clickable anchor from the heading text automatically, so don't add anchor markup yourself.

You'll see explicit anchor tags on existing pages, like:

```
## Heading text <a href="#heading-text" id="heading-text"></a>
```

> The migration added these to pin a stable anchor, so links to the heading keep working even if its text changes later. Don't add them to new headings, and leave the existing ones in place. If you reword a heading that already has one, keep its anchor tag so existing links don't break.

Author: ```suggestion
These pin a stable anchor, so links to the heading keep working even if its text changes later. It is not necessary to add them to new headings, but leave the existing ones in place. If you reword a heading that already has one, keep its anchor tag so existing links don't break.
```
